views/js/qtiCreator/tpl/forms/interactions/slider.tpl

```diff
@@ -1,11 +1,11 @@
 
 <div class="panel">
     <label for="lowerBound" class="spinner">{{__ "Lower Bound"}}</label>
-    <input name="lowerBound" value="{{lowerBound}}" data-increment="1" type="text" />
+    <input name="lowerBound" value="{{lowerBound}}" data-increment="1" data-min="0" validator="$integer" type="text" />
     <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
     <span class="tooltip-content">{{__ "The lower bound of the slider"}}</span>
     <label for="upperBound" class="spinner">{{__ "Upper Bound"}}</label>
-    <input name="upperBound" value="{{upperBound}}" data-increment="1" type="text" />
+    <input name="upperBound" value="{{upperBound}}" data-increment="1" data-min="0" validator="$integer" type="text" />
```

Comment:
Do you think we should use the integer validator on the step field as well?

Since we only use the integer responseType for this interaction, and any float value you put in as the step is automatically rounded down, I think it would make the UX better to give visual feedback when the user's input is invalid.

Reply:
Removed.